SophiApp/SophiApp/Resources/UIData.json

```diff
@@ -1595,11 +1595,11 @@
   {
     "Id": 344,
     "Header": {
-      "EN": "Folder for saving screenshots by pressing Win+PrtScn",
+      "EN": "Select folder to save screenshots when pressing Win+PrtScn",
       "RU": "Папка для сохранения скриншотов по нажатию Win+PrtScn"
     },
     "Description": {
-      "EN": "The function will be applied only if you choose to remove OneDrive. Otherwise the backup functionality for the \"Desktop\" and \"Pictures\" folders in OneDrive breaks",
+      "EN": "The function will be applied only if you have chosen to remove OneDrive. Otherwise the backup functionality for the \"Desktop\" and \"Pictures\" folders in OneDrive breaks.",
```

Comment:
I don't fully understand this. What exactly breaks? - How does OneDrive tie in?

Reply:
The problem is that if you OneDrive is installed, this feature will break the OneDrive backuping. So in the future we will need to check whether OneDrive is checked for uninstallation or not to let user change the folder for saving screenshots on the Desktop instead of Pictures.

Reply:
https://github.com/farag2/Sophia-Script-for-Windows/blob/6b23fcde50d7e6841afa078c964bb41a3c9c92bc/Sophia%20Script/Sophia%20Script%20for%20Windows%2010/Module/Sophia.psm1#L6655